docs/cpp/kind.rst

```diff
@@ -1,10 +1,16 @@
 Kind
 ====
 
+Every :cpp:class:`Term <cvc5::api::Term>` has a kind which represents its type, for example whether it is an equality (:cpp:enumerator:`EQUAL <cvc5::api::Kind::EQUAL>`), a conjunction (:cpp:enumerator:`AND <cvc5::api::Kind::AND>`), or a bitvector addition (:cpp:enumerator:`BITVECTOR_PLUS <cvc5::api::Kind::BITVECTOR_PLUS>`).
+
+Note that the api type :cpp:enum:`cvc5::api::Kind` roughly corresponds to :cpp:enum:`cvc5::Kind`, but is a different type and has a few subtle differences.
```

Comment:
```suggestion
Note that the API type :cpp:enum:`cvc5::api::Kind` roughly corresponds to :cpp:enum:`cvc5::Kind`, but is a different type and has a few subtle differences.
```

Reply:
I'm not sure if we should even mention that. `cvc5::Kind` should not be accessible from outside at all.